src/style.css

```
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
```

you can just use `:not(:last-child)`
```suggestion
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```